docs/source/external_sources/digital_collections.rst

```diff
@@ -0,0 +1,392 @@
+	end
+
+.. code-block::
+    :name: Copy all topical subjects
+    :caption: Copy all topical subjects
+
+	rule "Copy all topical subjects"
+		when
+			exist "/*[local-name()='record']/*[local-name()='metadata']/*[local-name()='mods']/*[local-name()='subject']/*[local-name()='topic']"
+		then
+			copy "/*[local-name()='record']/*[local-name()='metadata']/*[local-name()='mods']/*[local-name()='subject']/*[local-name()='topic']" to "dc"."subject"
+	end
+
+.. code-block::
+    :name: Copy all geographic subjects
+    :caption: Copy all geographic subjects
+
+	rule "Copy all geographic subjects"
+		when
+			exist "/*[local-name()='record']/*[local-name()='metadata']/*[local-name()='mods']/*[local-name()='subject']/*[local-name()='geographic']"
+		then
+			copy "/*[local-name()='record']/*[local-name()='metadata']/*[local-name()='mods']/*[local-name()='subject']/*[local-name()='geographic']" to "dcterms"."coverage"
+	end
+	
+.. code-block::
+    :name: Copy Creators
+    :caption: Copy Creators
+
+	rule "Copy all names as creators"
+		when
+			exist "/*[local-name()='record']/*[local-name()='metadata']/*[local-name()='mods']/*[local-name()='name']/*[local-name()='namePart']"
+		then
+			copy "/*[local-name()='record']/*[local-name()='metadata']/*[local-name()='mods']/*[local-name()='name']/*[local-name()='namePart']" to "dc"."creator"
+	end	
+
+.. code-block::
+    :name: Copy rights values
+    :caption: Copy rights values
+
+	rule "Copy rights values"
+		when
+			exist "/*[local-name()='record']/*[local-name()='metadata']/*[local-name()='mods']/*[local-name()='accessCondition']"
+		then 
+			copy "/*[local-name()='record']/*[local-name()='metadata']/*[local-name()='mods']/*[local-name()='accessCondition']" to "dcterms"."accessRights"
+	end
+
+.. code-block::
+    :name: Set discovery resource type of Digital Collections Item
+    :caption: Set discovery resource type of Digital Collections Item
+
+	rule "Set discovery resource type of Digital Collections Item"
+    	when
+        	not exist "//*[local-name()='typeOfResource'][@collection='yes']"
+    	then
+        	set "digital_items" in "discovery"."resourceType"
+	end
+
+.. code-block::
+    :name: Set discovery resource type of Digital Collection
+    :caption: Set discovery resource type of Digital Collection
+
+	rule "Set discovery resource type of Digital Collection"
+    	when
+         	exist "/*[local-name()='record']/*[local-name()='metadata']/*[local-name()='mods']/*[local-name()='typeOfResource'][@collection='yes']"   
+    	then
+        	set "digital_collection" in "discovery"."resourceType"
+	end
+	
+RegEx for Links
+===============
+
+Thumbnail: ^(http(s):\/\/).*(TN\/view)$
+
+Object (Item/Collection): ^(http(s):\/\/digital\.lib\.utk\.edu\/collections\/islandora\/object\/).*([0-9]|collections%3A.*|gsmrc%3A.*|arrowmont%3A.*)$
```

Comment:
Same here:

```
Object (Item/Collection): 

.. code-block::

    ^(http(s):\/\/digital\.lib\.utk\.edu\/collections\/islandora\/object\/).*([0-9]|collections%3A.*|gsmrc%3A.*|arrowmont%3A.*)$


```

Reply:
@markpbaggett - thanks for this suggestion. I've added code blocks for both regular expressions.